src/languages/en.ts

```diff
@@ -2050,6 +2050,11 @@ export default {
             updateCurrencyPrompt: 'It looks like your Workspace is currently set to a different currency than USD. Please click the button below to update your currency to USD now.',
             updateToUSD: 'Update to USD',
         },
+        connection: {
+            error: {
+                genericUpdate: `Unexpected error while updating the field, please try again later`,
```

Comment:
Should we have a period at the end of these?

Reply:
I copied this from an existing text for `report.genericUpdateReportFieldFailureMessage` 😄 The existing text doesn't contain a period, so I decided to go without a period

https://github.com/Expensify/App/blob/00e0121fa17a7c19335a8fa10eed51c25e933d6a/src/languages/en.ts#L2293

Reply:
Sounds good. I think we should probably standardise whether or not errors have periods, but we can do that outside this PR. I created an issue here: https://github.com/Expensify/App/issues/38989